src/AzureAppConfigurationImpl.ts

```diff
@@ -14,6 +14,7 @@ import { RefreshTimer } from "./refresh/RefreshTimer";
 import { CorrelationContextHeaderName } from "./requestTracing/constants";
 import { createCorrelationContextHeader, requestTracingEnabled } from "./requestTracing/utils";
 import { KeyFilter, LabelFilter, SettingSelector } from "./types";
+import { featureFlagsKeyName, featureManagementKeyName } from "./featureManagement/constants";

```

Comment:
Can we make the casing consistent with other constants? For example, CorrelationContextHeaderName is also a constant, but the casing is different.

Reply:
Sure, that's also the thing I want to do. Constants should be in`UPPER_SNAKE_CASE`, but in SDK it's `camelCase` starting with lowercase which is the same as normal variable names. 

Anyway, I prefer to use UPPER_SNAKE_CASE honoring the convention. I'll do that in another PR.

Reply:
unify constant naming in https://github.com/Azure/AppConfiguration-JavaScriptProvider/pull/68

Reply:
merged from main branch, fixing the naming issue. Now other constants are all UPPER_SNAKE_CASE